examples/north_sea/generate_mesh.py

```diff
@@ -0,0 +1,66 @@
+import qmesh
+
+
+def generate_mesh(level):
+    """
+    Generate a mesh of the North Sea domain using qmesh.
+    """
+    name = "north_sea.{:s}"
+
+    # Setup qmesh
+    qmesh.setLogOutputFile("qmesh.log")
+    qmesh.initialise()
+
+    # Read shapefile describing domain boundaries
+    boundary = qmesh.vector.Shapes()
+    boundary.fromFile("data/Boundary.shp")
```

Comment:
Maybe the mesh generation related files could go to a separate subdir to differentiate from forcing data.